### PR TITLE
Content updates on the Community page

### PR DESCRIPTION
**Pull request by znegrin**
I just updated the Community page according to the [balsamiq draft](https://balsamiq.cloud/sdjen2c/pedp6ry/r8F6B) on the [issue 1404](https://github.com/src-d/backlog/issues/1404) from the backlog. Please let me know if there is something wrong. I guess the Events photo could be better as I just took it from the mockup.

Thank you!

image

### CODE REVIEW

File: hugo/layouts/community/single.html [hunk 1]
```diff
@@ -5,92 +5,30 @@
...
+                <div class="card light">
                     <div class="card-body p-5">
-                        <a href="https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM" class="ic icon-slack text-light"></a>
-                        <h4 class="title-line line-yellow m-0">Join Our Community</h4>
-                        <p>Join hundreds of developers passionate about Code as Data and Machine Learning on Code</p>
-                        <form action="">
-                            <!-- <input type="text" class="form-control mb-3" placeholder="Your Email Address"> -->
-                            <button type="button" onclick="location.href='https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM';" class="btn btn-highlight w-100">Join Us Now</button>
-                        </form>
+                        <a href="https://forum.sourced.tech" class="text-dark">
```
Comment (dpordomingo) [inline hugo/layouts/community/single.html]: I'm not sure if it clear enough if there's any actionable part in the right box. I see the code, so I know that the whole box is a link... but not sure about the UX.
image

**Comment by znegrin** [comment]
Hi @vcoisne,

I uploaded all the requested changes.

image

Thank you!

**Comment by vcoisne** [comment]
thanks LGTM cc @dpordomingo

File: hugo/layouts/community/single.html [hunk 2]
```diff
@@ -23,6 +23,7 @@ <h2 class="title-line line-yellow">Our Community Channels</h2>
                         <a href="https://forum.sourced.tech" class="text-dark">
                             <h3 class="title-line line-yellow">source{d} Forum</h3>
                             <p>This is the forum where users and community members discuss anything source{d} related. source{d} is the data platform for the software development life cycle. Save time and modernize your codebase by gaining visibility into your software portfolio, engineering team and processes.</p>
+                            <a class="btn btn-highlight col-md-6 mt-3" href="https://forum.sourced.tech">Forum</a>
```
Comment (dpordomingo) [inline hugo/layouts/community/single.html]: There's a link inside of the link, what is not HTML well-formed.
Now that we have clear CTA, I'd remove the outer link,

**Comment by dpordomingo** [comment]
We also have a problem with Forum title vertical alignment
image

**Comment by znegrin** [comment]
> We also have a problem with Forum title vertical alignment
> image

@dpordomingo I think you are missing something, because right now it should look like this:

image

**Comment by znegrin** [comment]
> LGTM, but broken HTML code.
> We can merge once that issue is fixed.

Hi @dpordomingo, I uploaded a fix for the anchor. Thank you!

**Comment by vcoisne** [comment]
Changes look good. We're missing the Discourse logo/icon though?

Screen Shot 2019-06-25 at 10 15 35 AM

**Comment by znegrin** [comment]
> Changes look good. We're missing the Discourse logo/icon though?
> 
> Screen Shot 2019-06-25 at 10 15 35 AM

Maybe it is something related to the cache? It looks fine to me on staging:

image

**Comment by vcoisne** [comment]
@znegrin indeed, may bad. That page is ready to ship.

**Comment by dpordomingo** [comment]
This branch has conflicts that must be resolved.
Please update it considering the current state of `src-d:revamp-candidate`

**Comment by znegrin** [comment]
I think it should be fine now.

**Comment by znegrin** [comment]
> Not yet :(
> I think it would help if you rebase over `src-d:revamp-candidate` or pull it against your branch, solve the conflicts, commit, and then push again.

Ok, I think now it should be fine. Thanks!

**Pull request merged**